index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mobile Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <section id="nav">
+    <div class="nav-bar">
+      <div class="menu-bar"></div>
+      </div>
+  </section>
+
+  <section id="hero">
+    <div clsss="container">
+      <div class="image-container"></div>
+    </div>
+    <div class="hero-text">
+      <h1>
+        Hey There. <br>I’m Elias. <br> <span>I’m a software developer</span>
+      </h1>
+   
+      <p>
+        I can help you build a product, <br>feature or website Look through <br>some
+        of my work and experience! <br>If you like what you see and have <br>a project you need coded,<br>
+        don’t hesitate to contact me.
+      </p>
+    </div>
+  </section>
+
+  <section id="footer">
+    <div class="social-links">
+      <ul>
+        <li><a href="#"><img src="twitter (1).svg"></a></li>
+        <li><a href="#"><img src="Linkedin icon.svg"></a></li>
+        <li><a href="#"><img src="medium.svg"></a></li>
+        <li><a href="#"><img src="github.svg"></a></li>
+        <li><a href="#"><img src="angellist.svg"></a></li>
```

Comment:
- [optional] Please, don't forget to add value to the images `alt` attributes. 👍 The required `alt` attribute specifies an alternate text for an image, if the image cannot be displayed. ℹ️